pinot-minion/src/main/java/com/linkedin/pinot/minion/MinionStarter.java

```diff
@@ -55,8 +55,7 @@
 
   private HelixAdmin _helixAdmin;
 
-  public MinionStarter(String zkAddress, String helixClusterName, Configuration config)
-      throws Exception {
+  public MinionStarter(String zkAddress, String helixClusterName, Configuration config) throws Exception {
```

Comment:
Not a blocker for this PR, but we need to address the code formatting change due to IDE upgrade.